assets/app/lib/settings.rb

```diff
@@ -5,6 +5,15 @@
 module Lib
   module Settings
     DARK = `window.matchMedia('(prefers-color-scheme: dark)').matches`.freeze
+    # http://mkweb.bcgsc.ca/colorblind/ 12 color palette
+    ROUTE_COLORS = %i[#A40122 #008DF9 #00FCCF #FF5AAF].freeze
+
+    routes = {}
```

Comment:
can we again not use lower case for constants, and similarly do

ROUTES = ROUTE_COLERS.flat_map.with_index do |color, n|
 [["r#{n}_color, color]...]
end.to_h

something like that